index.html

```diff
@@ -1,111 +1,181 @@
+       <div class="catalog-item">
+        <p class="label">New</p>
+        <img src="" alt="" class="catalog-image">
+        <p class="catalog-price">$199</p>
+        <h3 class="product-name">Name product</h3>
+       </div>
+       <div class="catalog-item">
+        <p class="label">New</p>
+        <img src="" alt="" class="catalog-image">
+        <p class="catalog-price">$199</p>
+        <h3 class="product-name">Name Product</h3>
+       </div>
+       <div class="catalog-item">
+        <p class="label">Sold out</p>
+        <img src="" alt="" class="catalog-image">
+        <p class="catalog-price">$199</p>
+        <h3 class="product-name">Name Product</h3>
+       </div>
+       <div class="catalog-item">
+        <p class="label">New</p>
+        <img src="" alt="" class="catalog-image">
+        <p class="catalog-price">$199</p>
+        <h3 class="product-name">Name Product</h3>
+       </div>
+       <div class="catalog-item">
+        <p class="label">New</p>
+        <img src="" alt="" class="catalog-image">
+        <p class="catalog-price">$199</p>
+        <h3 class="product-name">Name Product</h3>
+      </div>
+      </article>
+    </section>
+    <section class="our-blog">
+      <h2 class="section-name">Our blog</h2>
+      <article class="sonic-live">
+        <p class="label">NFT</p>
+        <img src="" alt="big-face" class="big-face-image">
+        <p class="artist-name">Tony Nguyen</p>
+        <time datetime="2022-06-20" class="time">20 Jun 2022</time>
+        <h3 class="article-name">Sonic live</h3>
+        <p class="article-content">One September 22, 2021 A volutpat nulla eros, nulla ultrices nec nulla imperdiet...</p>
+        <a href="" class="read-more">Read more</a>
+      </article>
+      <a href="" class="read-more">Read more</a>
+    </section>
+    <section class="about-us">
+       <h2 class="section-name">About us</h2>
+       <div class="about-us-text">
+        <p class="text">Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla partiatur. Excepteur sint occae cat cupidatat non proident, sunt in culpa qui officia dese runt mollit anim id est labourm velit esse cillum odlore eu fugiat nulla pariatu epreur sint occaecat</p>
+       </div>
+      <div class="high-quality">
+        <h3 class="slills-name">High quality</h3>
+        <p class="skills-description">Lorem Ipsum in simply dummy text of the printing and Ipsum typessetting industry.</p>
+       </div>
+       <div class="big-collection">
+        <h3 class="skills-name">Big collection</h3>
+        <p class="skills-description">Lorem Ipsum in simply dummy text of the printing and Ipsum typessetting industry.</p>
+       </div>
+       <div class="top-resourse">
+        <h3 class="skills-name">Top resourse</h3>
+        <p class="skills-description">Lorem Ipsum in simply dummy text of the printing and Ipsum typessetting industry.</p>
+       </div>
+       <div class="Big-community">
```

Comment:
No Uppercase letter in class name please - Big-community ==> big-community

Reply:
done